praw/models/reddit/subreddit.py

```diff
@@ -1517,6 +1524,96 @@ def update_invite(self, redditor, permissions=None):
         self.subreddit._reddit.post(url, data=data)
 
 
+class Modmail(object):
+    """Provides modmail functions for a subreddit."""
+    def __call__(self,  # pylint: disable=invalid-name,redefined-builtin
+                 id=None, url=None):
+        """Get an individual conversation."""
+        return ModmailConversation(self.subreddit._reddit, id=id, url=url)
+
+    def __init__(self, subreddit):
+        self.subreddit = subreddit
+
+
+class ModmailConversation(RedditBase):
+    """A class for modmail conversations."""
+    STR_FIELD = 'id'
+
+    @classmethod
+    def parse(cls, data, reddit):
+        if data['conversation']:
```

Comment:
Ah. Conversation is always set. I was using [Message.parse](https://github.com/praw-dev/praw/blob/f84eae4bc8f5048f044d6d1619e27a2fe7b3ee82/praw/models/reddit/message.py) as a model, but now that I think about it those conditions are accomplishing something else.